GitDiffMargin/Core/IMarginCore.cs

```diff
@@ -29,6 +25,9 @@ internal interface IMarginCore
         double EditorChangeLeft { get; }
         double EditorChangeWidth { get; }
         double ScrollChangeWidth { get; }
+        event EventHandler BrushesChanged;
+
+        event EventHandler<HunksChangedEventArgs> HunksChanged;
```

Comment:
💭 I can't figure out what's going on with the event order and grouping. This is not a style I'm used to seeing. If you want to keep elements in a particular order, consider using an order that works with StyleCop Analyzers (in particular [SA1201](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1201.md) which has a default but can be configured).